pkg/operator/ceph/file/mds/mds_liveness_probe.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
```

Comment:
Let's note for the future why we aren't using `set -e` or `set -u` for this script.

```suggestion
#!/usr/bin/env bash
# do not use 'set -e -u` etc. because it is important to only fail this probe when failure is certain
# spurious failures risk destabilizing ceph or the filesystem

```